### ISSUE TITLE
Screen Reader Support Specification Adaptation - Rating Indicator Component

### ISSUE THREAD

**Issue by elenastoyanovaa** [SAP/ui5-webcomponents#2665]
**Is your feature request related to a problem? Please describe.**
We should analyse the existing screen reader support implementation, along with the UI5 SR Specification itself and Aria Best Practices and adapt them in a way that fit the ui5 web components.

**Describe the solution you'd like**
Analyse, create a proposal specification for the rating indicator component. The screen reader support implementation proposal, should be documented in the accessibility status wiki of the component.

**Comment by elenastoyanovaa** [comment]
The existing screen reader support is checked and some enhancements are needed. They will be processed in https://github.com/SAP/ui5-webcomponents/issues/2688
